spec/classes/jenkins_slave_spec.rb

```diff
@@ -37,6 +37,16 @@
       let(:params) { {:enable => false, :ensure => 'stopped' } }
       it { should contain_service('jenkins-slave').with(:enable => false, :ensure => 'stopped') }
     end
+
+    describe 'with single tool location' do
+      let(:params) { { :tool_locations => 'Python-2.7:/usr/bin/python2.7' } }
+      it { should contain_file(slave_runtime_file).with_content(/--toolLocation Python-2.7=\/usr\/bin\/python2.7/) }
+    end
+
+    describe 'with multiple tool locations' do
+      let(:params) { { :tool_locations => 'Python-2.7:/usr/bin/python2.7 Java-1.8:/usr/bin/java' } }
+      it { should contain_file(slave_runtime_file).with_content(/--toolLocation Python-2.7=\/usr\/bin\/python2.7|--toolLocation Java-1.8=\/usr\/bin\/java/) }
```

Comment:
This doesn't test that both option string are present.  How about something along these lines?

```
it do
  should contain_file(slave_runtime_file).
    with_content(/--toolLocation Python-2.7=\/usr\/bin\/python2.7/).
    with_content(/--toolLocation Java-1.8=\/usr\/bin\/java/)
end
```